packages/cli/cli/src/cli.ts

```diff
@@ -618,6 +620,34 @@ function addFdrCommand(cli: Argv<GlobalCliOptions>, cliContext: CliContext) {
     );
 }
 
+function addOpenAPIFdrCommand(cli: Argv<GlobalCliOptions>, cliContext: CliContext) {
+    cli.command(
+        "openapi-fdr <path-to-output>",
```

Comment:
nit: lets call this `fdr v2`

Reply:
actually thoughts on just passing in a `--v2` command to `fern fdr fdr.json` and if that flag is passed in then it delegates to your library?